data/homepage.yaml

```diff
@@ -165,3 +135,37 @@ links:
   platform:
     home: https://www.redhat.com/en/technologies/management/ansible
     use_cases: https://www.redhat.com/en/technologies/management/ansible/use-cases
+welcome:
+  title: Welcome to the new Ansible.com!
+  intro: Welcome home and thank you for supporting Ansible. View the video to learn about what changed and why...
+  url: https://www.youtube.com/embed/zHrIo4E1HmY?si=aXMEyR14P_6gKoj2
+catalyst:
+  section_icon: /assets/images/orchestration-cyan.svg
+  icon_alt: Ansible Catalyst program icon
+  title: Ansible Catalyst
+  intro: This is a new formed program lead by Red Hat to spark additional user interaction, spotlight the ecosystems contributions and share insights on driving automation adoption at your organizations.
+  articles:
+    collection_spotlight:
+      heading: Collection Spotlight
+      description: Every week we will spotlight a new collection available to the Ansible ecosystem.
```

Comment:
Again with the word "spotlight". It's somewhat overloaded here. It's fine in the heading but I would suggest using a synonym in the description.